### DISCUSSION TITLE
Wrong school listed, cannot change

### DISCUSSION THREAD

**Discussion by boiledtech-dev** [community/community#35255] [GitHub Education]
Hello all, when I imputed my email address, it autofilled another similarly named school in another country. I cannot undo this autofill. It also is not allowing me to go forward with my application as the school is in another country.

please help

**Comment by samus-aran** [comment]
Hi @boiledtech-dev 👋 🌞 
We'd love to help! Are you able to provide us more explicit feedback on your draft application. This might help someone give you a nudge in the right direction.

- What is the domain of the email address?
- What school is it incorrectly linking it to?
- What school do you wish it to be correctly linked to?

(apologies on the earlier edit 😅 )
